src/main/java/com/liveramp/hyperminhash/Murmur3.java

```diff
@@ -0,0 +1,544 @@
+/*
```

Comment:
I lifted this from apache Hive, we could use it for hashing if we decide we don't want any dependencies

Reply:
I'm adding a NOTICE file